index.html

```diff
@@ -2554,12 +2553,13 @@ <h3>Using computed attribute values</h3>
           of its descendant <code>&lt;div&gt;</code> elements that do correspond to <a>Script Events</a>
           are different to what they would be if that intermediate <code>&lt;div&gt;</code> element's <code>begin</code>
           time were ignored.</p>
-        <p class="ednote">MAYBE REMOVE Or, also for example, if styling is used in a TTML document, the computed value of the <code>tts:fontSize</code>
+        <aside class="example">An example of this relating to functionality not required by DAPT is styling.
+          If styling is used in a TTML document, the computed value of the <code>tts:fontSize</code>
           attribute when expressed as a percentage is relative to the computed value of the same attribute
           on the element's parent.
           Furthermore, in this example, the algorithm for generating ISDs (Intermediate Synchronic Documents)
           involves reparenting some elements. The computed value that applies is the one <em>before</em>
-          rehoming has taken place.</p>
+          rehoming has taken place.</aside>
```

Comment:
Do we really need this example? It is about a "functionality not required by DAPT". I would remove it altogether. If not, I would rephrase saying something like:
> If a DAPT document contains additional features not required by DAPT such as styling and the DAPT document is processed by a processor supporting these features, support for inheritance of the inheritable features such as tts:fontSize, is required, per TTML2.

Reply:
Removing it was one of the options before, but until nobody came down on the side of "remove". Not wanting to leave an editorial note in, if avoidable, I just turned it into an example. I don't mind deleting it if it isn't helpful.

Reply:
Is my rephrasing not acceptable?

Reply:
Sorry, I understood your preference to be for removal, and didn't focus on the rephrasing. Reading it again, I think the following two paragraphs already say the same thing as the rephrased version - maybe we should just add to one of them the example of `tts:fontSize` to make the abstract statements more concrete, and remove the example?

Concretely, I'm suggesting the paragraph:

>The semantics defined by [[TTML2](https://pr-preview.s3.amazonaws.com/w3c/dapt/pull/256.html#bib-ttml2)] or, for vocabulary defined herein, this specification, take precedence in this scenario. Implementations MUST compute attribute values based on the contents of the [document instance](https://www.w3.org/TR/ttml2/#terms-document-instance) before applying those computed values to DAPT data model objects.

would become:

>The semantics defined by [[TTML2](https://pr-preview.s3.amazonaws.com/w3c/dapt/pull/256.html#bib-ttml2)] or, for vocabulary defined herein, this specification, take precedence in this scenario. Implementations MUST compute attribute values based on the contents of the [document instance](https://www.w3.org/TR/ttml2/#terms-document-instance) before applying those computed values to DAPT data model objects. **For example a processor that supports TTML2 styling features would need to implement the TTML2 semantics for inheritance and computing relative values of attributes like `tts:fontSize`.**

Reply:
and remove the example?

Reply:
Yes, and remove the example.

Reply:
Done in 5617ab0.